package.json

```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "@module-federation/utilities": "^1.7.0",
-    "@theforeman/vendor": "^13.1.0",
+    "@theforeman/vendor": "^14.0.0",
```

Comment:
If we update this, we'll also need to merge https://github.com/theforeman/foreman/pull/10272, right? It's not that we can merge this without updating to PF5.

Reply:
Eh, but https://github.com/theforeman/foreman-js/pull/481 is still not merged which I assumed was a prerequisite for PF5? Where does that leave us?

Reply:
vendor 14 doesnt include pf5

Reply:
Ah, so we'll have version 15 soon after? That's the reason why I had preferred to get rid of foreman-js before making big stack upgrades but I guess that's too late now.